internal/services/formatters/hcl/checkov/formatter.go

```diff
@@ -90,6 +91,15 @@ func (f *Formatter) parseOutput(output string) error {
 	return nil
 }
 
+// nolint:lll // const ansi is a regex and cannot be break into more lines
+func (f *Formatter) removeAnsiCharacters(output string) []byte {
+	const ansi = "[\u001B\u009B][[\\]()#;?]*(?:(?:(?:[a-zA-Z\\d]*(?:;[a-zA-Z\\d]*)*)?\u0007)|(?:(?:\\d{1,4}(?:;\\d{0,4})*)?[\\dA-PRZcf-ntqry=><~]))"
+
+	re := regexp.MustCompile(ansi)
```

Comment:
What do you think about declaring this regex globally? Using globally we avoid recompiling this regex each time that the formatter runs.

Reply:
i don't think this would make a difference since the formatter will run just once in a project scan and i apply this just once on the tool docker result

Reply:
I don't think that this is always true because an user can set multiple [work directories](https://docs.horusec.io/docs/tutorials/how-to-run-horusec-multiples-directories/)

Reply:
is it really worth it to make something this little global just to avoid some corner case like using work dir?

Reply:
I don't see any problem with global variables that are constant, I even prefer it, I believe it's easier to read. I think it's up to you to make it global or not, this change really doesn't have any negative or positive impact.